www/search.php

```diff
@@ -889,25 +889,25 @@ function WildCardQuery($stype, $Like, $query) {
-		<OPTION VALUE="<?php echo SEARCH_FIELD_DEPENDS_BUILD        . '"'; if ($stype == SEARCH_FIELD_DEPENDS_BUILD)        echo 'SELECTED'; ?>>Depends Build</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_DEPENDS_LIB          . '"'; if ($stype == SEARCH_FIELD_DEPENDS_LIB)          echo 'SELECTED'; ?>>Depends Lib</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_DEPENDS_RUN          . '"'; if ($stype == SEARCH_FIELD_DEPENDS_RUN)          echo 'SELECTED'; ?>>Depends Run</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_DEPENDS_ALL          . '"'; if ($stype == SEARCH_FIELD_DEPENDS_ALL)          echo 'SELECTED'; ?>>Depends Build/Lib/Run</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_LATEST_LINK          . '"'; if ($stype == SEARCH_FIELD_LATEST_LINK)          echo 'SELECTED'; ?>>Latest Link</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_LONGDESCRIPTION      . '"'; if ($stype == SEARCH_FIELD_LONGDESCRIPTION)      echo 'SELECTED'; ?>>Long Description</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_LICENSE_PERMS        . '"'; if ($stype == SEARCH_FIELD_LICENSE_PERMS)        echo 'SELECTED'; ?>>LICENSE_PERMS</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_LICENSE_RESTRICTED   . '"'; if ($stype == SEARCH_FIELD_LICENSE_RESTRICTED)   echo 'SELECTED'; ?>>_LICENSE_RESTRICTED</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_MAKEFILE             . '"'; if ($stype == SEARCH_FIELD_MAKEFILE)             echo 'SELECTED'; ?>>Makefile (ports only)</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_MAINTAINER           . '"'; if ($stype == SEARCH_FIELD_MAINTAINER)           echo 'SELECTED'; ?>>Maintainer</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_MANUAL_PACKAGE_BUILD . '"'; if ($stype == SEARCH_FIELD_MANUAL_PACKAGE_BUILD) echo 'SELECTED'; ?>>MANUAL_PACKAGE_BUILD</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_MESSAGEID            . '"'; if ($stype == SEARCH_FIELD_MESSAGEID)            echo 'SELECTED'; ?>>Message ID</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_PACKAGE              . '"'; if ($stype == SEARCH_FIELD_PACKAGE)              echo 'SELECTED'; ?>>Package Name</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_PKG_PLIST            . '"'; if ($stype == SEARCH_FIELD_PKG_PLIST)            echo 'SELECTED'; ?>>pkg-plist</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_NAME                 . '"'; if ($stype == SEARCH_FIELD_NAME)                 echo 'SELECTED'; ?>>Port Name</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_SHORTDESCRIPTION     . '"'; if ($stype == SEARCH_FIELD_SHORTDESCRIPTION)     echo 'SELECTED'; ?>>Short Description</OPTION>
-		<OPTION VALUE="<?php echo SEARCH_FIELD_PATHNAME             . '"'; if ($stype == SEARCH_FIELD_PATHNAME)             echo 'SELECTED'; ?>>Under a pathname</OPTION>
+		<OPTION VALUE="<?php echo SEARCH_FIELD_COMMITMESSAGE        . '"'; if ($stype == SEARCH_FIELD_COMMITMESSAGE)        echo ' SELECTED'; ?>>Commit Message</OPTION>
```

Comment:
Why is a leading space important there?

Reply:
![image](https://user-images.githubusercontent.com/1646397/115162584-c5b82000-a0e7-11eb-8a29-5b57cdac0e30.png)
Without the space the attributes run into eachother on the selected element. It's not _important_, per se, since browsers seem to figure it out anyway, but that's why it's a nit. 😅  (sorry for high dpi screenshot)